client/src/actions/surveyActions.test.js

```diff
@@ -0,0 +1,10 @@
+import { getSurvey, setSurveyLoading } from "./surveyActions";
+import { GET_SURVEY, SURVEY_LOADING } from "../actions/types";
+
+describe("Survey Actions", () => {
+  it("can return SURVEY_LOADING ACTION", () => {
```

Comment:
with test names, make them more assertive e.g. `it("does this thing")` instead of `it("can do this thing")` or `it("should do this thing")`